src/test/java/com/zaxxer/nuprocess/DirectWriteTest.java

```diff
@@ -105,7 +107,9 @@ public void onStdout(ByteBuffer buffer, boolean closed)
 
       NuProcessBuilder pb = new NuProcessBuilder(processListener, command);
       NuProcess nuProcess = pb.start();
-      for (int i = 0; i < 1000; i++) {
+      // TODO: given a large i (e.g. 1,000, 10,000), this unit test (testConsecutiveWrites) will
+      //       produce a side-effect on InterruptTest (has problem on Mac OS X, but works on Linux and Win32).
+      for (int i = 0; i < 100; i++) {
```

Comment:
If the ``reuseForks=false`` does address the issue, I would say commit the pom.xml and bump the iterations here back up to 1000.